src/main/java/no/nav/tag/tiltaksgjennomforing/datavarehus/AvroTiltakHendelseFabrikk.java

```diff
@@ -76,7 +76,7 @@ public static AvroTiltakHendelse konstruer(Avtale avtale, DvhHendelseType hendel
     }
 
     private Boolean erMaster(Avtale avtale) {
-        if (avtale.getTiltakstype() == Tiltakstype.SOMMERJOBB || avtale.getTiltakstype() == Tiltakstype.MIDLERTIDIG_LONNSTILSKUDD || avtale.getTiltakstype() == Tiltakstype.VARIG_LONNSTILSKUDD) {
+        if (avtale.getTiltakstype() == Tiltakstype.SOMMERJOBB || avtale.getTiltakstype() == Tiltakstype.MIDLERTIDIG_LONNSTILSKUDD || avtale.getTiltakstype() == Tiltakstype.VARIG_LONNSTILSKUDD || avtale.getTiltakstype() == Tiltakstype.ARBEIDSTRENING) {
```

Comment:
Denne begynner å bli litt stygg 😅 

Hva med noe ala:
```suggestion
Set<Tiltakstype> master = Set.of(...);
...
private Boolean erMaster(Avtale avtale) {
    return master.contains(tiltakstype));
}
```

Reply:
Enig, fikset!